templates/common/_base/files/clean-ovs-network.yaml

```diff
@@ -0,0 +1,60 @@
+mode: 0755
+path: "/usr/local/bin/clean-ovs.sh"
```

Comment:
Given how the scripts are closely connected (needing to know the same interface names, etc) it seems like it might be better to put the setup and cleanup both into a single file, and run it as either `configure-ovs.sh setup` or `configure-ovs.sh cleanup` depending on which mode you need?

Reply:
or, if you're going to merge https://github.com/openshift/cluster-network-operator/pull/782 into this, then maybe just `configure-ovs.sh {{.NetworkType}}`, where `configure-ovs.sh OpenShiftSDN` would delete `br-ex` if it existed, and `configure-ovs.sh OVNKubernetes` would delete `br0` if it existed and set up `br-ex`.

Reply:
fixed.